File: C++/MathematicalPrograms/2DShapes/AreaAndPerimeterOfSquare.cpp
#include <iostream>
#include <string>
#include <math.h>

using namespace std;

int main() {
	int side;
	double area, perimeter;
	cout << "Enter the Side measure";
	cin >> side;
	area = pow(side, 2);
	perimeter = 4*side;
	cout << "Area = " << area << " sq.cm.";
	cout << "\nPerimeter = " << perimeter << " cm";
}

File: Java/MathematicalPrograms/2DShapes/AreaAndPerimeterOfSquare.java
import java.util.Scanner;
import java.lang.Math;
public class AreaAndPerimeterOfSquare {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter the Side measure");
		int side = myObj.nextInt();
		double peri = 4*side;
		double area = Math.pow(side, 2);
		System.out.println("Area = " + area + " sq.cm.");
		System.out.println("Perimeter = " + peri + " cm.");
	}
}